gammapy/catalog/tests/test_hawc.py

```diff
@@ -0,0 +1,23 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+import pytest
+from ..hawc import SourceCatalog2HWC
+
+@pytest.fixture #TODO: add here a scope like in test_gammacat.py?
+def hwc2_fhl():
+    filename='$GAMMAPY_EXTRA/datasets/catalogs/2HWC.ecsv' #TODO: Not sure whether this makes sence, __init__ of SourceCatalog2HWC 
+                                                          # has this string as default argument
+    return SourceCatalog2HWC(filename = filename) 
```

Comment:
Run `pep8` on your files and fix up style violations like whitespace on this line.
See http://pep8.org/ if you're not familar.
I've been using PyCharm (free edition: https://www.jetbrains.com/pycharm-edu/download/ ) for a few years, and never had to run `pep8` from the command line. It's awesome, highly recommend it.

Reply:
The white spaces are only due to the comment at the end. 
Will delete it, when it is clear what to do here.